subsys/bluetooth/mesh/sar_cfg_srv.c

```diff
@@ -26,6 +26,22 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_mesh_sar_cfg_srv);
 
+static int sar_rx_store(struct bt_mesh_model *model, bool delete)
+{
+	const void *data = delete ? NULL : &bt_mesh.sar_rx;
+	size_t len = delete ? 0 : sizeof(struct bt_mesh_sar_rx);
+
+	return bt_mesh_model_data_store(model, false, "rx", data, len);
```

Comment:
Could the setting entries be more specified? Seems just `rx` and `tx` so general. `bt_mesh_model_data_store` is used to construct paths for the settings for any models (including vendor models that are out of our control). There is a very high probability somebody else will name proprietary settings as `rx` and `tx`.
Could it be `sar_rx` and `sar_tx`?

Reply:
Ok, I thought that it made a separate namespace per model, but if that is not the case I aggre that something more spesific would be better.

Reply:
_(just for clarification)_ `bt_mesh_model_data_store` stores model's data under `bt/mesh/<v/s>/<element><model_id>/<data>` path:
https://github.com/zephyrproject-rtos/zephyr/blob/93f7a6730d12dd5f8eae8cadb41adeb66c1136d9/subsys/bluetooth/mesh/access.c#L1968-L1972
https://github.com/zephyrproject-rtos/zephyr/blob/93f7a6730d12dd5f8eae8cadb41adeb66c1136d9/subsys/bluetooth/mesh/access.c#L1734-L1744
so the collision between models should never happen.

Reply:
Ah, ok then. Guess it does not make a huge difference either way.